utils/_context/_scenarios.py

```diff
@@ -1266,6 +1266,19 @@ class scenarios:
         """,
     )
 
+    appsec_api_security_rc = EndToEndScenario(
```

Comment:
Can you add this new scenario in https://github.com/DataDog/system-tests/blob/main/.github/workflows/ci.yml ?